bundles/org.openhab.binding.tuya/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
 					<option value="https://openapi.tuyaeu.com">Central Europe</option>
 					<option value="https://openapi-weaz.tuyaeu.com">Western Europe (Azure/MS)</option>
 					<option value="https://openapi.tuyain.com">India</option>
+					<option value="https://openapi-sg.iotbing.com">Singapore</option>
```

Comment:
Can you also [update](https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file) the I18N properties file?

https://github.com/openhab/openhab-addons/blob/c698d0c5aeca8e63c325915dd342fd66261139e3/bundles/org.openhab.binding.tuya/src/main/resources/OH-INF/i18n/tuya.properties#L23-L28